src/Main.elm

```diff
@@ -134,14 +164,30 @@ update msg model =
             let
                 ( subModel, subCmd ) =
                     Multiselect.update subscriptionEvent model.genresMultiselect
+
+                selectedGenres =
+                    subModel.selected
+                        |> Set.fromList
+
+                newUrl =
+                    case List.length subModel.selected of
+                        0 ->
+                            model.location.origin
+
+                        _ ->
+                            subModel.selected
+                                |> List.map (\( m, n ) -> m)
+                                |> String.join "+"
+                                |> (++)
+                                    (model.location.origin
+                                        ++ "?genres="
+                                    )
```

Comment:
This is a naïve approach that would kill off other query params, so I don't love it, but it does work. Nothing obvious comes to mind, but then again, I am very tired.